runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -1245,15 +1246,10 @@ TR::Register *J9::X86::TreeEvaluator::newEvaluator(TR::Node *node, TR::CodeGener
 
 TR::Register *J9::X86::TreeEvaluator::multianewArrayEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   static char *useDirectHelperCall = feGetEnv("TR_MultiANewArrayEvaluatorUseDirectCall");
-
    TR::Node *firstChild = node->getFirstChild();
    TR::Node *secondChild = node->getSecondChild();
    TR::Node *thirdChild = node->getThirdChild();
 
-   if (useDirectHelperCall || !secondChild->getOpCode().isLoadConst() || secondChild->getInt()!=2)
-      return TR::TreeEvaluator::performHelperCall(node, NULL, TR::acall, true, cg);
-
    // 2-dimensional MultiANewArray
    TR::Compilation *comp = cg->comp();
    TR_J9VMBase *fej9 = (TR_J9VMBase *)(comp->fe());
```

Comment:
This should use `static_cast`. Similarly in other places where we fixed the casts on the Z PR.

Reply:
Proper casts added in: 997900e